### DISCUSSION TITLE
Cannot change or hide backdrop for text entities

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#985] [General]
**Description**

When creating a `TextPrimitive` `SceneEntity`, there is a color field but no background_color. When the color is set to black (`{0, 0, 0, 1}`), the backdrop is automatically set to white. I want to make the backdrop transparent or a different color from white.

- Version: latest
- Platform: all

**Steps To Reproduce**
* Fail to create a TextPrimitive with black text and a non-white backdrop

**Expected Behavior**
* Succeed

**Comment by foxhubber** [comment]
Internal tracking ticket: [FG-6664](https://linear.app/foxglove/issue/FG-6664/cannot-change-or-hide-backdrop-for-text-entities-[foxgloveschemas-141])

**Comment by jtbandes** [comment]
Duplicate of #843
